Nuvoton: Support boot stack size configuration option

ccli8 opened the pull request:
### Description

Based on #8039, this PR adds support for target configuration option `target.boot-stack-size` for configuring interrupt stack size in mbed-os 5 and main stack size in mbed-os 2.

#### Support targets
- NUMAKER_PFM_NANO130
- NUMAKER_PFM_NUC472
- NUMAKER_PFM_M453
- NUMAKER_PFM_M487/NUMAKER_IOT_M487
- NUMAKER_PFM_M2351

### Pull request type

    [ ] Fix
    [ ] Refactor
    [ ] Target update
    [X] Functionality change
    [ ] Docs update
    [ ] Test update
    [ ] Breaking change

ccli8 committed:
[Nuvoton] Support boot stack size configuration option

ciarmcom commented:
@ccli8, thank you for your changes.
@ARMmbed/mbed-os-maintainers please review.

0xc0170 commented:
cc @mprse

mprse reviewed (approved):
The changes looks good. 

I'm also working on boot stack size (PR https://github.com/ARMmbed/mbed-os/pull/9092). Probably I will divide my PR and add commit for each Vendor. So I can remove Nuvoton from my PR since probably this one will be merged faster. 
I have one request. Can you provide results of the following tests:
 - tests-mbed_hal-stack_size_unification (https://github.com/ARMmbed/mbed-os/pull/9092/commits/5ffe000dd42a0b4f862b8e5e6df63150998b8de0)
 - tests-mbedmicro-rtos-mbed-heap_and_stack

ccli8 committed:
[NANO130] Change main thread stack size to default for stack size unification

NANO130 just has 16 KiB SRAM. User application may change it through "rtos.main-thread-stack-size"
configuration option.

ccli8 commented:
@mprse **NUMAKER_PFM_NANO130** cannot pass `tests-mbed_hal-stack_size_unification` because it configures main thread stack size to 3092. Now I remove the configuration in https://github.com/ARMmbed/mbed-os/pull/9102/commits/28b820fbf0575b0b7c927b31195fa9e55559dc12 for stack size unification.

ccli8 commented:
#### NANO130+ARM

```
mbedgt: test case report:
+-------------------------+---------------------+--------------------------------------------------+-------------------------------+--------+--------+--------+--------------------+
| target                  | platform_name       | test suite                                       | test case                     | passed | failed | result | elapsed_time (sec) |
+-------------------------+---------------------+--------------------------------------------------+-------------------------------+--------+--------+--------+--------------------+
| NUMAKER_PFM_NANO130-ARM | NUMAKER_PFM_NANO130 | mbed-os-tests-mbed_hal-stack_size_unification    | Stack size unification test   | 1      | 0      | OK     | 0.0                |
| NUMAKER_PFM_NANO130-ARM | NUMAKER_PFM_NANO130 | mbed-os-tests-mbedmicro-rtos-mbed-heap_and_stack | Test heap allocation and free | 1      | 0      | OK     | 0.02               |
| NUMAKER_PFM_NANO130-ARM | NUMAKER_PFM_NANO130 | mbed-os-tests-mbedmicro-rtos-mbed-heap_and_stack | Test heap in range            | 1      | 0      | OK     | 0.0                |
| NUMAKER_PFM_NANO130-ARM | NUMAKER_PFM_NANO130 | mbed-os-tests-mbedmicro-rtos-mbed-heap_and_stack | Test isr stack in range       | 1      | 0      | OK     | 0.0                |
| NUMAKER_PFM_NANO130-ARM | NUMAKER_PFM_NANO130 | mbed-os-tests-mbedmicro-rtos-mbed-heap_and_stack | Test main stack in range      | 1      | 0      | OK     | 0.0                |
+-------------------------+---------------------+--------------------------------------------------+-------------------------------+--------+--------+--------+--------------------+
mbedgt: test case results: 5 OK
```

#### NANO130+GCC_ARM

```
mbedgt: test case report:
+-----------------------------+---------------------+--------------------------------------------------+-------------------------------+--------+--------+--------+--------------------+
| target                      | platform_name       | test suite                                       | test case                     | passed | failed | result | elapsed_time (sec) |
+-----------------------------+---------------------+--------------------------------------------------+-------------------------------+--------+--------+--------+--------------------+
| NUMAKER_PFM_NANO130-GCC_ARM | NUMAKER_PFM_NANO130 | mbed-os-tests-mbed_hal-stack_size_unification    | Stack size unification test   | 1      | 0      | OK     | 0.0                |
| NUMAKER_PFM_NANO130-GCC_ARM | NUMAKER_PFM_NANO130 | mbed-os-tests-mbedmicro-rtos-mbed-heap_and_stack | Test heap allocation and free | 1      | 0      | OK     | 0.02               |
| NUMAKER_PFM_NANO130-GCC_ARM | NUMAKER_PFM_NANO130 | mbed-os-tests-mbedmicro-rtos-mbed-heap_and_stack | Test heap in range            | 1      | 0      | OK     | 0.0                |
| NUMAKER_PFM_NANO130-GCC_ARM | NUMAKER_PFM_NANO130 | mbed-os-tests-mbedmicro-rtos-mbed-heap_and_stack | Test isr stack in range       | 1      | 0      | OK     | 0.0                |
| NUMAKER_PFM_NANO130-GCC_ARM | NUMAKER_PFM_NANO130 | mbed-os-tests-mbedmicro-rtos-mbed-heap_and_stack | Test main stack in range      | 1      | 0      | OK     | 0.0                |
+-----------------------------+---------------------+--------------------------------------------------+-------------------------------+--------+--------+--------+--------------------+
mbedgt: test case results: 5 OK
```

#### NANO130+IAR

```
mbedgt: test case report:
+-------------------------+---------------------+--------------------------------------------------+-------------------------------+--------+--------+--------+--------------------+
| target                  | platform_name       | test suite                                       | test case                     | passed | failed | result | elapsed_time (sec) |
+-------------------------+---------------------+--------------------------------------------------+-------------------------------+--------+--------+--------+--------------------+
| NUMAKER_PFM_NANO130-IAR | NUMAKER_PFM_NANO130 | mbed-os-tests-mbed_hal-stack_size_unification    | Stack size unification test   | 1      | 0      | OK     | 0.0                |
| NUMAKER_PFM_NANO130-IAR | NUMAKER_PFM_NANO130 | mbed-os-tests-mbedmicro-rtos-mbed-heap_and_stack | Test heap allocation and free | 1      | 0      | OK     | 0.02               |
| NUMAKER_PFM_NANO130-IAR | NUMAKER_PFM_NANO130 | mbed-os-tests-mbedmicro-rtos-mbed-heap_and_stack | Test heap in range            | 1      | 0      | OK     | 0.0                |
| NUMAKER_PFM_NANO130-IAR | NUMAKER_PFM_NANO130 | mbed-os-tests-mbedmicro-rtos-mbed-heap_and_stack | Test isr stack in range       | 1      | 0      | OK     | 0.0                |
| NUMAKER_PFM_NANO130-IAR | NUMAKER_PFM_NANO130 | mbed-os-tests-mbedmicro-rtos-mbed-heap_and_stack | Test main stack in range      | 1      | 0      | OK     | 0.0                |
+-------------------------+---------------------+--------------------------------------------------+-------------------------------+--------+--------+--------+--------------------+
mbedgt: test case results: 5 OK
```

ccli8 commented:
#### NUC472+ARM

```
mbedgt: test case report:
+------------------------+--------------------+--------------------------------------------------+-------------------------------+--------+--------+--------+--------------------+
| target                 | platform_name      | test suite                                       | test case                     | passed | failed | result | elapsed_time (sec) |
+------------------------+--------------------+--------------------------------------------------+-------------------------------+--------+--------+--------+--------------------+
| NUMAKER_PFM_NUC472-ARM | NUMAKER_PFM_NUC472 | mbed-os-tests-mbed_hal-stack_size_unification    | Stack size unification test   | 1      | 0      | OK     | 0.0                |
| NUMAKER_PFM_NUC472-ARM | NUMAKER_PFM_NUC472 | mbed-os-tests-mbedmicro-rtos-mbed-heap_and_stack | Test heap allocation and free | 1      | 0      | OK     | 0.88               |
| NUMAKER_PFM_NUC472-ARM | NUMAKER_PFM_NUC472 | mbed-os-tests-mbedmicro-rtos-mbed-heap_and_stack | Test heap in range            | 1      | 0      | OK     | 0.0                |
| NUMAKER_PFM_NUC472-ARM | NUMAKER_PFM_NUC472 | mbed-os-tests-mbedmicro-rtos-mbed-heap_and_stack | Test isr stack in range       | 1      | 0      | OK     | 0.0                |
| NUMAKER_PFM_NUC472-ARM | NUMAKER_PFM_NUC472 | mbed-os-tests-mbedmicro-rtos-mbed-heap_and_stack | Test main stack in range      | 1      | 0      | OK     | 0.0                |
+------------------------+--------------------+--------------------------------------------------+-------------------------------+--------+--------+--------+--------------------+
mbedgt: test case results: 5 OK
```

#### NUC472+GCC_ARM

```
mbedgt: test case report:
+----------------------------+--------------------+--------------------------------------------------+-------------------------------+--------+--------+--------+--------------------+
| target                     | platform_name      | test suite                                       | test case                     | passed | failed | result | elapsed_time (sec) |
+----------------------------+--------------------+--------------------------------------------------+-------------------------------+--------+--------+--------+--------------------+
| NUMAKER_PFM_NUC472-GCC_ARM | NUMAKER_PFM_NUC472 | mbed-os-tests-mbed_hal-stack_size_unification    | Stack size unification test   | 1      | 0      | OK     | 0.0                |
| NUMAKER_PFM_NUC472-GCC_ARM | NUMAKER_PFM_NUC472 | mbed-os-tests-mbedmicro-rtos-mbed-heap_and_stack | Test heap allocation and free | 1      | 0      | OK     | 0.9                |
| NUMAKER_PFM_NUC472-GCC_ARM | NUMAKER_PFM_NUC472 | mbed-os-tests-mbedmicro-rtos-mbed-heap_and_stack | Test heap in range            | 1      | 0      | OK     | 0.0                |
| NUMAKER_PFM_NUC472-GCC_ARM | NUMAKER_PFM_NUC472 | mbed-os-tests-mbedmicro-rtos-mbed-heap_and_stack | Test isr stack in range       | 1      | 0      | OK     | 0.0                |
| NUMAKER_PFM_NUC472-GCC_ARM | NUMAKER_PFM_NUC472 | mbed-os-tests-mbedmicro-rtos-mbed-heap_and_stack | Test main stack in range      | 1      | 0      | OK     | 0.0                |
+----------------------------+--------------------+--------------------------------------------------+-------------------------------+--------+--------+--------+--------------------+
mbedgt: test case results: 5 OK
```

#### NUC472+IAR

```
mbedgt: test case report:
+------------------------+--------------------+--------------------------------------------------+-------------------------------+--------+--------+--------+--------------------+
| target                 | platform_name      | test suite                                       | test case                     | passed | failed | result | elapsed_time (sec) |
+------------------------+--------------------+--------------------------------------------------+-------------------------------+--------+--------+--------+--------------------+
| NUMAKER_PFM_NUC472-IAR | NUMAKER_PFM_NUC472 | mbed-os-tests-mbed_hal-stack_size_unification    | Stack size unification test   | 1      | 0      | OK     | 0.0                |
| NUMAKER_PFM_NUC472-IAR | NUMAKER_PFM_NUC472 | mbed-os-tests-mbedmicro-rtos-mbed-heap_and_stack | Test heap allocation and free | 1      | 0      | OK     | 0.61               |
| NUMAKER_PFM_NUC472-IAR | NUMAKER_PFM_NUC472 | mbed-os-tests-mbedmicro-rtos-mbed-heap_and_stack | Test heap in range            | 1      | 0      | OK     | 0.0                |
| NUMAKER_PFM_NUC472-IAR | NUMAKER_PFM_NUC472 | mbed-os-tests-mbedmicro-rtos-mbed-heap_and_stack | Test isr stack in range       | 1      | 0      | OK     | 0.0                |
| NUMAKER_PFM_NUC472-IAR | NUMAKER_PFM_NUC472 | mbed-os-tests-mbedmicro-rtos-mbed-heap_and_stack | Test main stack in range      | 1      | 0      | OK     | 0.0                |
+------------------------+--------------------+--------------------------------------------------+-------------------------------+--------+--------+--------+--------------------+
mbedgt: test case results: 5 OK
```

ccli8 commented:
#### M453+ARM

```
mbedgt: test case report:
+----------------------+------------------+--------------------------------------------------+-------------------------------+--------+--------+--------+--------------------+
| target               | platform_name    | test suite                                       | test case                     | passed | failed | result | elapsed_time (sec) |
+----------------------+------------------+--------------------------------------------------+-------------------------------+--------+--------+--------+--------------------+
| NUMAKER_PFM_M453-ARM | NUMAKER_PFM_M453 | mbed-os-tests-mbed_hal-stack_size_unification    | Stack size unification test   | 1      | 0      | OK     | 0.0                |
| NUMAKER_PFM_M453-ARM | NUMAKER_PFM_M453 | mbed-os-tests-mbedmicro-rtos-mbed-heap_and_stack | Test heap allocation and free | 1      | 0      | OK     | 0.02               |
| NUMAKER_PFM_M453-ARM | NUMAKER_PFM_M453 | mbed-os-tests-mbedmicro-rtos-mbed-heap_and_stack | Test heap in range            | 1      | 0      | OK     | 0.0                |
| NUMAKER_PFM_M453-ARM | NUMAKER_PFM_M453 | mbed-os-tests-mbedmicro-rtos-mbed-heap_and_stack | Test isr stack in range       | 1      | 0      | OK     | 0.0                |
| NUMAKER_PFM_M453-ARM | NUMAKER_PFM_M453 | mbed-os-tests-mbedmicro-rtos-mbed-heap_and_stack | Test main stack in range      | 1      | 0      | OK     | 0.0                |
+----------------------+------------------+--------------------------------------------------+-------------------------------+--------+--------+--------+--------------------+
mbedgt: test case results: 5 OK
```

#### M453+GCC_ARM

```
mbedgt: test case report:
+--------------------------+------------------+--------------------------------------------------+-------------------------------+--------+--------+--------+--------------------+
| target                   | platform_name    | test suite                                       | test case                     | passed | failed | result | elapsed_time (sec) |
+--------------------------+------------------+--------------------------------------------------+-------------------------------+--------+--------+--------+--------------------+
| NUMAKER_PFM_M453-GCC_ARM | NUMAKER_PFM_M453 | mbed-os-tests-mbed_hal-stack_size_unification    | Stack size unification test   | 1      | 0      | OK     | 0.0                |
| NUMAKER_PFM_M453-GCC_ARM | NUMAKER_PFM_M453 | mbed-os-tests-mbedmicro-rtos-mbed-heap_and_stack | Test heap allocation and free | 1      | 0      | OK     | 0.01               |
| NUMAKER_PFM_M453-GCC_ARM | NUMAKER_PFM_M453 | mbed-os-tests-mbedmicro-rtos-mbed-heap_and_stack | Test heap in range            | 1      | 0      | OK     | 0.0                |
| NUMAKER_PFM_M453-GCC_ARM | NUMAKER_PFM_M453 | mbed-os-tests-mbedmicro-rtos-mbed-heap_and_stack | Test isr stack in range       | 1      | 0      | OK     | 0.0                |
| NUMAKER_PFM_M453-GCC_ARM | NUMAKER_PFM_M453 | mbed-os-tests-mbedmicro-rtos-mbed-heap_and_stack | Test main stack in range      | 1      | 0      | OK     | 0.0                |
+--------------------------+------------------+--------------------------------------------------+-------------------------------+--------+--------+--------+--------------------+
mbedgt: test case results: 5 OK
```

#### M453+IAR

```
mbedgt: test case report:
+----------------------+------------------+--------------------------------------------------+-------------------------------+--------+--------+--------+--------------------+
| target               | platform_name    | test suite                                       | test case                     | passed | failed | result | elapsed_time (sec) |
+----------------------+------------------+--------------------------------------------------+-------------------------------+--------+--------+--------+--------------------+
| NUMAKER_PFM_M453-IAR | NUMAKER_PFM_M453 | mbed-os-tests-mbed_hal-stack_size_unification    | Stack size unification test   | 1      | 0      | OK     | 0.0                |
| NUMAKER_PFM_M453-IAR | NUMAKER_PFM_M453 | mbed-os-tests-mbedmicro-rtos-mbed-heap_and_stack | Test heap allocation and free | 1      | 0      | OK     | 0.02               |
| NUMAKER_PFM_M453-IAR | NUMAKER_PFM_M453 | mbed-os-tests-mbedmicro-rtos-mbed-heap_and_stack | Test heap in range            | 1      | 0      | OK     | 0.0                |
| NUMAKER_PFM_M453-IAR | NUMAKER_PFM_M453 | mbed-os-tests-mbedmicro-rtos-mbed-heap_and_stack | Test isr stack in range       | 1      | 0      | OK     | 0.0                |
| NUMAKER_PFM_M453-IAR | NUMAKER_PFM_M453 | mbed-os-tests-mbedmicro-rtos-mbed-heap_and_stack | Test main stack in range      | 1      | 0      | OK     | 0.0                |
+----------------------+------------------+--------------------------------------------------+-------------------------------+--------+--------+--------+--------------------+
mbedgt: test case results: 5 OK
```

ccli8 commented:
#### M487+ARM

```
mbedgt: test case report:
+----------------------+------------------+--------------------------------------------------+-------------------------------+--------+--------+--------+--------------------+
| target               | platform_name    | test suite                                       | test case                     | passed | failed | result | elapsed_time (sec) |
+----------------------+------------------+--------------------------------------------------+-------------------------------+--------+--------+--------+--------------------+
| NUMAKER_PFM_M487-ARM | NUMAKER_PFM_M487 | mbed-os-tests-mbed_hal-stack_size_unification    | Stack size unification test   | 1      | 0      | OK     | 0.0                |
| NUMAKER_PFM_M487-ARM | NUMAKER_PFM_M487 | mbed-os-tests-mbedmicro-rtos-mbed-heap_and_stack | Test heap allocation and free | 1      | 0      | OK     | 0.04               |
| NUMAKER_PFM_M487-ARM | NUMAKER_PFM_M487 | mbed-os-tests-mbedmicro-rtos-mbed-heap_and_stack | Test heap in range            | 1      | 0      | OK     | 0.0                |
| NUMAKER_PFM_M487-ARM | NUMAKER_PFM_M487 | mbed-os-tests-mbedmicro-rtos-mbed-heap_and_stack | Test isr stack in range       | 1      | 0      | OK     | 0.0                |
| NUMAKER_PFM_M487-ARM | NUMAKER_PFM_M487 | mbed-os-tests-mbedmicro-rtos-mbed-heap_and_stack | Test main stack in range      | 1      | 0      | OK     | 0.0                |
+----------------------+------------------+--------------------------------------------------+-------------------------------+--------+--------+--------+--------------------+
mbedgt: test case results: 5 OK
```

#### M487+GCC_ARM

```
mbedgt: test case report:
+--------------------------+------------------+--------------------------------------------------+-------------------------------+--------+--------+--------+--------------------+
| target                   | platform_name    | test suite                                       | test case                     | passed | failed | result | elapsed_time (sec) |
+--------------------------+------------------+--------------------------------------------------+-------------------------------+--------+--------+--------+--------------------+
| NUMAKER_PFM_M487-GCC_ARM | NUMAKER_PFM_M487 | mbed-os-tests-mbed_hal-stack_size_unification    | Stack size unification test   | 1      | 0      | OK     | 0.0                |
| NUMAKER_PFM_M487-GCC_ARM | NUMAKER_PFM_M487 | mbed-os-tests-mbedmicro-rtos-mbed-heap_and_stack | Test heap allocation and free | 1      | 0      | OK     | 0.05               |
| NUMAKER_PFM_M487-GCC_ARM | NUMAKER_PFM_M487 | mbed-os-tests-mbedmicro-rtos-mbed-heap_and_stack | Test heap in range            | 1      | 0      | OK     | 0.0                |
| NUMAKER_PFM_M487-GCC_ARM | NUMAKER_PFM_M487 | mbed-os-tests-mbedmicro-rtos-mbed-heap_and_stack | Test isr stack in range       | 1      | 0      | OK     | 0.0                |
| NUMAKER_PFM_M487-GCC_ARM | NUMAKER_PFM_M487 | mbed-os-tests-mbedmicro-rtos-mbed-heap_and_stack | Test main stack in range      | 1      | 0      | OK     | 0.0                |
+--------------------------+------------------+--------------------------------------------------+-------------------------------+--------+--------+--------+--------------------+
mbedgt: test case results: 5 OK
```

#### M487+IAR

```
mbedgt: test case report:
+----------------------+------------------+--------------------------------------------------+-------------------------------+--------+--------+--------+--------------------+
| target               | platform_name    | test suite                                       | test case                     | passed | failed | result | elapsed_time (sec) |
+----------------------+------------------+--------------------------------------------------+-------------------------------+--------+--------+--------+--------------------+
| NUMAKER_PFM_M487-IAR | NUMAKER_PFM_M487 | mbed-os-tests-mbed_hal-stack_size_unification    | Stack size unification test   | 1      | 0      | OK     | 0.0                |
| NUMAKER_PFM_M487-IAR | NUMAKER_PFM_M487 | mbed-os-tests-mbedmicro-rtos-mbed-heap_and_stack | Test heap allocation and free | 1      | 0      | OK     | 0.03               |
| NUMAKER_PFM_M487-IAR | NUMAKER_PFM_M487 | mbed-os-tests-mbedmicro-rtos-mbed-heap_and_stack | Test heap in range            | 1      | 0      | OK     | 0.0                |
| NUMAKER_PFM_M487-IAR | NUMAKER_PFM_M487 | mbed-os-tests-mbedmicro-rtos-mbed-heap_and_stack | Test isr stack in range       | 1      | 0      | OK     | 0.0                |
| NUMAKER_PFM_M487-IAR | NUMAKER_PFM_M487 | mbed-os-tests-mbedmicro-rtos-mbed-heap_and_stack | Test main stack in range      | 1      | 0      | OK     | 0.0                |
+----------------------+------------------+--------------------------------------------------+-------------------------------+--------+--------+--------+--------------------+
mbedgt: test case results: 5 OK
```

ccli8 commented:
#### M2351+ARMC6

Currently, only ARMC6 is supported on M2351.

```
mbedgt: test case report:
+-------------------------+-------------------+--------------------------------------------------+-------------------------------+--------+--------+--------+--------------------+
| target                  | platform_name     | test suite                                       | test case                     | passed | failed | result | elapsed_time (sec) |
+-------------------------+-------------------+--------------------------------------------------+-------------------------------+--------+--------+--------+--------------------+
| NUMAKER_PFM_M2351-ARMC6 | NUMAKER_PFM_M2351 | mbed-os-tests-mbed_hal-stack_size_unification    | Stack size unification test   | 1      | 0      | OK     | 0.06               |
| NUMAKER_PFM_M2351-ARMC6 | NUMAKER_PFM_M2351 | mbed-os-tests-mbedmicro-rtos-mbed-heap_and_stack | Test heap allocation and free | 1      | 0      | OK     | 0.13               |
| NUMAKER_PFM_M2351-ARMC6 | NUMAKER_PFM_M2351 | mbed-os-tests-mbedmicro-rtos-mbed-heap_and_stack | Test heap in range            | 1      | 0      | OK     | 0.06               |
| NUMAKER_PFM_M2351-ARMC6 | NUMAKER_PFM_M2351 | mbed-os-tests-mbedmicro-rtos-mbed-heap_and_stack | Test isr stack in range       | 1      | 0      | OK     | 0.07               |
| NUMAKER_PFM_M2351-ARMC6 | NUMAKER_PFM_M2351 | mbed-os-tests-mbedmicro-rtos-mbed-heap_and_stack | Test main stack in range      | 1      | 0      | OK     | 0.05               |
+-------------------------+-------------------+--------------------------------------------------+-------------------------------+--------+--------+--------+--------------------+
mbedgt: test case results: 5 OK
```

ccli8 commented:
@mprse As above, I list the test results on Nuvoton targets.

mprse commented:
> @mprse As above, I list the test results on Nuvoton targets.

Looks great! Thanks for all this changes and test results. I will remove Nuvoton changes from my PR when this one is merged.
@0xc0170  I think we can start CI.

0xc0170 commented:
CI started

deepikabhavnani commented:
> @mprse **NUMAKER_PFM_NANO130** cannot pass `tests-mbed_hal-stack_size_unification` because it configures main thread stack size to 3092. Now I remove the configuration in [28b820f](https://github.com/ARMmbed/mbed-os/commit/28b820fbf0575b0b7c927b31195fa9e55559dc12) for stack size unification.

After changing the stack size of main thread, other test might fail in CI of NANO130, we do not have this hardware in CI so please test this locally

cmonr commented:
@mprse @ccli8 Looks like I missed this comment: https://github.com/ARMmbed/mbed-os/pull/9102#issuecomment-448307058

Please open an issue if problems with this PR are found. If the issues end up being severe, we can revert this PR.
